src/index.scss

```diff
@@ -8,3 +10,7 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 main {
   display: block;
 }
+
+$feature-flags: (
```

Comment:
The feature-flags should come before the stylesheet import See https://www.carbondesignsystem.com/tutorial/react/step-1#add-ui-shell